src/bootstrap/defaults/bootstrap.dist.toml

```diff
@@ -7,8 +7,6 @@ test-stage = 2
 doc-stage = 2
 # When compiling from source, you usually want all tools.
 extended = true
-# Use libtest built from the source tree instead of the precompiled one from stage 0.
-compiletest-use-stage0-libtest = false
```

Comment:
This reminds me that I forgot to update the docs for `compiletest-use-stage0-libtest` in `bootstrap.example.toml`; feel free to do that in this PR if you want.